mlir/test/Conversion/EmulateFp8ExtTrunc/Integration/roundtrip.mlir

```diff
@@ -67,32 +96,45 @@ func.func @main() {
 
   %id1 = arith.constant 1 : i64
   %in1 = arith.constant -0.0 : f32
-  // CHECK: 1, 0
+  // F8E5M2FNUZ: 1, 0
+  // F8E4M3FNUZ: 1, 0
+  // F8E5M2:     1, -0
+  // F8E4M3FN:   1, -0
   call @testTruncExt(%id1, %in1) : (i64, f32) -> ()
 
   %id2 = arith.constant 2 : i64
   %in2 = arith.constant 0x7f800000  : f32
-  // CHECK: 2, nan
+  // F8E5M2FNUZ: 2, nan
+  // F8E4M3FNUZ: 2, nan
+  // F8E5M2:     2, inf
+  // F8E4M3FN:   2, nan
+  //                384?
   call @testTruncExt(%id2, %in2) : (i64, f32) -> ()
 
   %id3 = arith.constant 3 : i64
   // COM: Largest e5m2
   %in3 = arith.constant 57344.0 : f32
   // F8E5M2FNUZ: 3, 57344
   // F8E4M3FNUZ: 3, 240
+  // F8E5M2:     3, 57344
+  // F8E4M3FN:   3, nan
   call @testTruncExt(%id3, %in3) : (i64, f32) -> ()
 
   %id4 = arith.constant 4 : i64
   %in4 = arith.constant -57344.0 : f32
   // F8E5M2FNUZ: 4, -57344
   // F8E4M3FNUZ: 4, -240
+  // F8E5M2:     4, -57344
+  // F8E4M3FN:   4, -nan
```

Comment:
Shouldn't it be `-inf`?

Reply:
OCP E4M3 doesn't have inf, and uses nan instead.

Reply:
... Ah, misread the test